modules/serverless-functions-secrets-configmaps-alternative-invocations.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Alternative ways to invoke the interactive prompt
+
+There is a number of more specific invocations of the `func config` utility. Use them to skip steps when navigating the `func config` operation tree:
+
+* `func config envs`
+** List configured environment variables:
++
+[source,terminal]
+----
+$ func config envs [-p <path>]
+----
+
+** Add environment variables to the function configuration:
```

Comment:
rather than making these sub-bullets I'd maybe just have them be their own steps - should the different `kn func config` steps maybe be separate modules / procedures?

Reply:
This section is supposed to be more of a reference one, it only needs to list the available commands. The commands themselves are intuitive enough after reading the previous sections, so they don't need anything more than a short description. Definitely no need for separate procedures.